server/fleet/app.go

```diff
@@ -371,3 +375,120 @@ type LicenseInfo struct {
 	// Note is any additional terms of license
 	Note string `json:"note,omitempty"`
 }
+
+type Logging struct {
+	Debug           bool              `json:"debug"`
+	Json            bool              `json:"json"`
+	ResultLogPlugin string            `json:"osquery_result_log_plugin"`
+	StatusLogPlugin string            `json:"osquery_status_log_plugin"`
+	FileSystem      *FileSystemConfig `json:"filesystem,omitempty"`
+	Firehose        *FirehoseConfig   `json:"firehose,omitempty"`
+	Kinesis         *KinesisConfig    `json:"kinesis,omitempty"`
+	Lambda          *LambdaConfig     `json:"lambda,omitempty"`
+	PubSub          *PubSubConfig     `json:"pubsub,omitempty"`
+}
+
+type FileSystemConfig struct {
```

Comment:
Nit: Let's keep the capitalization consistent
```suggestion
type FilesystemConfig struct {
```